entwatch/ze_danmaku_v1_5.cfg

```diff
@@ -1,136 +0,0 @@
-        "shortname"         "Aura"
-        "color"             "{white}"
-        "buttonclass"       "func_button"
-        "filtername"        ""
-        "hasfiltername"     "false"
-        "blockpickup"       "false"
-        "allowtransfer"     "true"
-        "forcedrop"         "true"
-        "chat"              "true"
-        "hud"               "true"
-        "hammerid"          "556397"
-        "mode"              "2"
-        "maxuses"           "0"
-        "cooldown"          "70"
-        "maxamount"         "1"
-    }
-    "3"
-    {
-        "name"              "Repel"
-        "shortname"         "Repel"
-        "color"             "{lightgreen}"
-        "buttonclass"       "func_button"
-        "filtername"        ""
-        "hasfiltername"     "false"
-        "blockpickup"       "false"
-        "allowtransfer"     "true"
-        "forcedrop"         "true"
-        "chat"              "true"
-        "hud"               "true"
-        "hammerid"          "878075"
-        "mode"              "2"
-        "maxuses"           "0"
-        "cooldown"          "60"
-        "maxamount"         "1"
-    }
-    "4"
-    {
-        "name"              "Ordnance"
-        "shortname"         "Ordnance"
-        "color"             "{red}"
-        "buttonclass"       "func_button"
-        "filtername"        ""
-        "hasfiltername"     "false"
-        "blockpickup"       "false"
-        "allowtransfer"     "false"
-        "forcedrop"         "false"
-        "chat"              "true"
-        "hud"               "true"
-        "hammerid"          "823017"
-        "mode"              "2"
-        "maxuses"           "0"
-        "cooldown"          "65"
-        "maxamount"         "1"
-    }
-    "5"
-    {
-        "name"              "Demonic Aura"
-        "shortname"         "Aura"
-        "color"             "{purple}"
-        "buttonclass"       "func_button"
-        "filtername"        ""
-        "hasfiltername"     "false"
-        "blockpickup"       "false"
-        "allowtransfer"     "false"
-        "forcedrop"         "false"
-        "chat"              "true"
-        "hud"               "true"
-        "hammerid"          "832248"
-        "mode"              "2"
-        "maxuses"           "0"
-        "cooldown"          "85"
-        "maxamount"         "1"
-    }
-    "6"
-    {
-        "name"              "Manipulation"
-        "shortname"         "Manipulation"
-        "color"             "{lightgreen}"
-        "buttonclass"       "func_button"
-        "filtername"        ""
-        "hasfiltername"     "false"
-        "blockpickup"       "false"
-        "allowtransfer"     "false"
-        "forcedrop"         "false"
-        "chat"              "true"
-        "hud"               "true"
-        "hammerid"          "913731"
-        "mode"              "2"
-        "maxuses"           "0"
-        "cooldown"          "55"
-        "maxamount"         "1"
-    }
-}
```

Comment:
Is there a reason for removing this config? We don't have the map on the server but in the rare case of some fixed version coming out I'd want this to fall back on.